frontend/packages/ceph-storage-plugin/src/components/independent-mode/install.tsx

```diff
@@ -27,6 +30,10 @@ const ERROR: DataState = {
 
 const getErrorText = (text: string) => <span className="im-install-page--error">{text}</span>;
 
+const mapDispatchToProps = (dispatch: Dispatch): MapDispatchToProps => ({
+  enableDashboard: () => dispatch(setFlag(OCS_INDEPENDENT_FLAG, true)),
```

Comment:
So after creating the cluster, we have dashboard enabled ?
If I got right, why this change ?